impl.go

```diff
@@ -21,7 +21,8 @@ import (
 )
 
 var (
-	flagSrcDir = flag.String("dir", "", "package source directory, useful for vendored code")
+	flagSrcDir     = flag.String("dir", "", "package source directory, useful for vendored code")
+	flagNoComments = flag.Bool("no-comments", false, "disable copying comments from the interface")
```

Comment:
I'd rather not have a double negative. Let's make this `-comments`, with a default of `true`.